packages/web/src/components/payout-wallet-display/PayoutWalletDisplay.tsx

```diff
@@ -38,22 +41,32 @@ const PayoutWalletDisplaySkeleton = () => {
 export const PayoutWalletDisplay = () => {
   const user = useSelector(getAccountUser)
   const payoutWallet = user?.spl_usdc_payout_wallet
+  const { cornerRadius } = useTheme()
 
   const { data: externalWalletOwner, isLoading: isLoadingOwner } =
     useWalletOwner(payoutWallet)
 
+  const isExternalWallet = !!payoutWallet
+
+  const iconComponent = useMemo(() => {
```

Comment:
this doesn't need to be memo'd it's actually less efficient to do so iirc. nit though